Worktez/worktez
# Fixed the UI of mail templates

### Functionality:
Fixed and improved mail template UI.

### Solution:
Changes in html file of mail templates.

### Risk level:

### How to test:
1.Open pulled code 
2. Go to mailer folder
3. use preview to see the html template.
